src/actions/CommandAction.ts

```diff
@@ -67,24 +69,45 @@ export default class CommandAction implements Action {
             await sleep(args.preCommandSleep);
           }
 
-          const [updatedOriginalSelections, updatedTargetSelections] =
-            await callFunctionAndUpdateSelections(
-              this.graph.rangeUpdater,
-              () => commands.executeCommand(args.command!, args.commandArgs),
-              editor.document,
-              [originalSelections, targetSelections]
-            );
+          const commandPromise = commands.executeCommand(
+            args.command!,
+            args.commandArgs
+          );
```

Comment:
I don't think you want to start the command before you call the track selections function. If it does an update before it yields you may miss the update. Could just be getting lucky that it hasn't hit you here

Reply:
Got it

Reply:
I actually had reservations about that. Updated now.